python/fledge/services/core/api/auth.py

```diff
@@ -342,6 +349,12 @@ async def get_user(request):
                 u["accessMethod"] = row["access_method"]
                 u["realName"] = row["real_name"]
                 u["description"] = row["description"]
+                if row["block_until"]:
+                    curr_time = datetime.datetime.now(datetime.timezone.utc).strftime(DATE_FORMAT)
+                    block_time = row["block_until"].split('.')[0] # strip time after HH:MM:SS for display
+                    if datetime.datetime.strptime(row["block_until"], DATE_FORMAT) > datetime.datetime.strptime(curr_time, DATE_FORMAT):
+                        u["status"] = "blocked"
+                        u["blockUntil"] = block_time
```

Comment:
As these both keys are dependent and only exists if block_until. So we may add either with
```
"block": {"status": true/false, "until": "YYY-MM-DD...."}

Or

only with single key "blockUntil"
```